config.yaml

```diff
@@ -37,3 +37,11 @@ options:
       Ref: https://grafana.com/docs/loki/latest/configure/#limits_config
     type: int
     default: 15
+  retention-period:
+    description: |
+      Sets a global retention period for log streams in Loki. A value of 0 disables retention (default).
+      Minimum retention period is 24h.
+      Specify the period in hours. For example, to set a 48-hour retention period, use `48`.
+      Specifying retention periods for individual streams is not currently supported.
+    type: int
+    default: 0
```

Comment:
I thought Loki's default is 30 days.
If so, then to remain consistent on upgrade, we would need to set if there.

Reply:
default is that they live forever cause `retention_enabled` in compactor should be set to True, if set to true and no value is provided or set to `0` then 744h (30days) retention is used.